File: 1. Two Sum/ans.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        for(int i = 0; i < nums.Length; i++) {
                for(int pi = i + 1; pi < nums.Length; pi++) {
                    if(nums[i] + nums[pi] == target) return new int[] {i,pi};
                }
            }
        return Array.Empty < int > ();
    }
}

File: 1. Two Sum/ans.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        #grabs the length of the nums list given
        nI = len(nums)

        #runs a for loop for x value at beginning of list
        for x in range(0, nI):
            for y in range(x + 1, nI):
                if nums[x] + nums[y] == target:
                    ans = [x, y]
                    return ans

